ChangeLog

```diff
@@ -1,3 +1,8 @@
+2017-08-04    <adam.johansen@gmail.com>
```

Comment:
Not to worry. I _finally_ started a repo dotfiles for just those types of things across machines.